torch/csrc/jit/codegen/cuda/lower2device.h

```diff
@@ -14,6 +14,18 @@
 #include <memory>
 #include <ostream>
 
+namespace torch {
+namespace jit {
+namespace fuser {
+namespace cuda {
+namespace kir {
+class ExprBuilderVisitor;
+}
+} // namespace cuda
+} // namespace fuser
+} // namespace jit
+} // namespace torch
+
```

Comment:
nit: We could just have `namespace kir { class ExprBuilderVisitor; }` inside the cuda namespace available below. That would make it a little clearer that it shares the same top-level namespace.